manifests/overlays/additional/kustomization.yaml

```diff
@@ -17,7 +17,6 @@ resources:
   - runtime-minimal-imagestream-beta.yaml
   - runtime-pytorch-imagestream-beta.yaml
   - runtime-rocm-pytorch-imagestream-beta.yaml
-  - runtime-rocm-tensorflow-imagestream-beta.yaml
```

Comment:
I guess I would've only removed this single line; or maybe commented it out, but, ok

/lgtm